grepper_python/__init__.py

```diff
@@ -0,0 +1,40 @@
+"""
+Python Grepper API
+~~~~~~~~~~~~~~~~~~~
+An API wrapper for the Grepper API.
+"""
+
+__title__ = "grepper-python"
+__author__ = "CodeGrepper"
+__license__ = "MIT"
+__copyright__ = "Copyright 2010-2023 Grepper, Inc."
+__version__ = "0.0.1a"
+
+__path__ = __import__("pkgutil").extend_path(__path__, __name__)
+
+import logging
+from typing import NamedTuple, Literal
+
+from .answer import GrepperAnswer
+from .main import Grepper
+
+
+class VersionInfo(NamedTuple):
```

Comment:
From what I've seen, you never used this class. From my experience, `__version__` is enough to be loaded from other software like python packing

Reply:
Thanks for the feedback!

Reply:
Of course ! I will do a more in depth review tomorrow probably